src/실전문제/시각.js

```diff
@@ -0,0 +1,16 @@
+const solution = (n) => {
+  let count = 0;
+
+  for (let h = 0; h <= n; h++) {
+    for (let m = 0; m < 60; m++) {
+      for (let s = 0; s < 60; s++) {
+        if (`${h}${m}${s}`.includes("3")) {
+          count += 1;
```

Comment:
```suggestion
  for (let h = 0; h <= n; h++) {
    for (let m = 0; m < 60; m++) {
      for (let s = 0; s < 60; s++) {
        if (`${h}${m}${s}`.includes("3")) {
          count++;
```

h++, m++, s++ 과 일관되게 count++ 은 어떤지 한번 제안드려봅니다 :sweat_smile: